src/IssueViz.Security/OpenInIDE/ListenerFactory.cs

```diff
@@ -73,5 +73,20 @@ HttpListener IListenerFactory.Create(int startPort, int endPort)
             }
             return listener;
         }
+
+        private static IDisposable CreateListener(int port)
+        {
+            var appProperties = new AppProperties(new Dictionary<string, object>())
```

Comment:
Owin provides strongly-typed wrapper classes to make working with the untyped property bags easier.